### PR TITLE
Updates to sliver workshop

### PR DESCRIPTION
**Pull request by Piinks**
This makes a few changes based on rehearsal feedback.

### CODE REVIEW

File: dartpad_codelabs/src/getting_started_with_slivers/step_05/solution.dart [hunk 1]
```diff
@@ -111,53 +113,54 @@ class WeeklyForecastList extends StatelessWidget {
 // Below this line are helper classes and data.
 
 const String baseAssetURL = 'https://dartpad-workshops-io2021.web.app/getting_started_with_slivers/';
```
Comment (darrenaustin) [inline dartpad_codelabs/src/getting_started_with_slivers/step_05/solution.dart:115]: This is a minor style thing and may just be me, but it looks weird to have the trailing '/' on the URL as then the usage of `${baseAssetURL}assets/foobar.jpeg` looks like it is missing a '/' 😄.

**Review by darrenaustin — APPROVED** [review]
Nice. LGTM.

**Comment by Piinks** [comment]
I have one more color tweak to make, don't land this yet :)

**Comment by Piinks** [comment]
Per @johnpryan on last update to this workshop (https://github.com/flutter/codelabs/pull/102), this is not covered by CI, so waiting on the checks is moot. I am going to merge so I can do another run through. :)

**Pull request merged**